doc/source/data/examples/pytorch_resnet_batch_prediction.ipynb

```diff
@@ -396,20 +412,27 @@
     "        with torch.inference_mode():\n",
     "            prediction = self.model(torch_batch)\n",
     "            predicted_classes = prediction.argmax(dim=1).detach().cpu()\n",
-    "            predicted_labels = [self.weights.meta[\"categories\"][i] for i in predicted_classes]\n",
-    "            return {\"predicted_label\": predicted_labels, \"original_image\": batch[\"original_image\"]}"
+    "            predicted_labels = [\n",
+    "                self.weights.meta[\"categories\"][i] for i in predicted_classes\n",
+    "            ]\n",
+    "            return {\n",
+    "                \"predicted_label\": predicted_labels,\n",
+    "                \"original_image\": batch[\"original_image\"],\n",
+    "            }\n"
    ]
   },
   {
    "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Then we use the {meth}`map_batches <ray.data.Dataset.map_batches>` API to apply the model to the whole dataset. \n",
+    "Then we use the [`map_batches`](https://docs.ray.io/en/latest/data/api/doc/ray.data.Dataset.map_batches.html) API to apply the model to the whole dataset.\n",
```

Comment:
we should keep these references as relative links right?

Reply:
Oh yep, good catch.